automated install/basic-install.sh

```diff
@@ -267,6 +267,51 @@ setStaticIPv4() {
 	fi
 }
 
+setDNS(){
+	DNSChoseCmd=(whiptail --separate-output --radiolist "Select DNS Servers" $r $c 2)
+	DNSChooseOptions=(Google "Use Google's DNS Servers" on
+					  DynDNS "Use DynDNS's DNS Servers" off)
+	DNSchoices=$("${DNSChoseCmd[@]}" "${DNSChooseOptions[@]}" 2>&1 >/dev/tty)
+
+	case $DNSchoices in
+		Google)
+			echo "Google selected."
+			piholeDNS1="8.8.8.8"
+			piholeDNS2="8.8.4.4"
+			;;
+		DynDNS)
+			echo "DynDNS selected."
+			piholeDNS1="208.67.222.222"
+			piholeDNS2="208.67.220.220"
+			;;
+	esac
+}
+
+versionCheckDNSmasq(){
+	# Check if /etc/dnsmasq.conf is from pihole.  If so replace with an original and install new in .d directory
+	dnsFile1="/etc/dnsmasq.conf"
+	dnsFile2="/etc/dnsmasq.conf.orig"
+	dnsSearch="addn-hosts=/etc/pihole/gravity.list"
+	
+	# Check dnsmasq.conf for pihole magic
+	if grep -q $dnsSearch $dnsFile1; then
```

Comment:
The function checkForDependencies() is run before versionCheckDNSmasq() in installPihole() and there for we can be certain that dnsmasq.conf that comes with the package is installed.  If not, we have other issues with the install. 

GNU had this about the -s switch:

> 
-s
--no-messages
Suppress error messages about nonexistent or unreadable files. Portability note: unlike GNU grep, 7th Edition Unix grep did not conform to POSIX, because it lacked -q and its -s option behaved like GNU grep's -q option.1 USG-style grep also lacked -q but its -s option behaved like GNU grep's. Portable shell scripts should avoid both -q and -s and should redirect standard and error output to /dev/null instead. (-s is specified by POSIX.)
>

 If you would like, we can wrap that grep with something like this:
```bash
if [ -d "/etc/dnsmasq.conf" ]; then
		grep statement
	else
		we are missing dnsmasq package
fi 
```
or pipe the errors to /dev/null